src/features/user/ManageProjects/ProjectsContainer.tsx

```diff
@@ -65,7 +65,9 @@ function SingleProject({ project }: { project: Properties }) {
               Number(project.countPlanted),
               1
             )}{' '}
-            {tCommon('tree', { count: Number(project.countPlanted) })}
+            {project.unitType === 'm2'
+              ? tCommon(project.unitType)
+              : tCommon('tree', { count: Number(project.countPlanted) })}
           </p>
```

Comment:
1. I suggest simply using something like
`tCommon(project.unitType, {formattedCount, count})` 
instead of handling different unitTypes in different ways. Make sure to not affect places that use the existing translation keys - you probably need new keys.

2. API needs to be updated to return `countPlanted` / `areaConserved` etc. for `unitType = 'm2'`. This should consider conservation projects as well. Discuss this with @jmiridis

3. Show the total units value if unitType is 'trees' or 'm2'.

4. Update the type of project to reflect the API response. Be specific about the values `unitType` can take.